embassy-stm32/src/can/bxcan/mod.rs

```diff
@@ -154,7 +155,10 @@ impl<T: Instance> Drop for CanConfig<'_, T> {
 
 /// CAN driver
 pub struct Can<'d, T: Instance> {
-    peri: PeripheralRef<'d, T>,
+    _peri: PeripheralRef<'d, T>,
+    instance: &'d crate::pac::can::Can,
```

Comment:
`crate::pac::can::Can` is already a pointer. don't store a pointer to it, store it owned directly. see #3018

Reply:
Ok, thanks.